portal/src/main/scala/cromwell/pipeline/datastorage/dto/User.scala

```diff
@@ -12,5 +17,27 @@ final case class User(
   profilePicture: Option[ProfilePicture] = None
 )
 
+object User {
+  implicit lazy val userFormat: OFormat[User] = Json.format[User]
+}
+
 final case class UserId(value: String) extends MappedTo[String]
+
+object UserId {
+  implicit lazy val userIdFormat: OFormat[UserId] = Json.format[UserId]
+}
+
 final case class ProfilePicture(value: Array[Byte]) extends MappedTo[Array[Byte]]
+
+object ProfilePicture {
+  implicit lazy val profilePictureFormat: OFormat[ProfilePicture] = Json.format[ProfilePicture]
```

Comment:
Since you have custom `Reads` and `Writes`, you do not need this `Format`

Reply:
removed, thanks for explaination